src/tlo/methods/labour.py

```diff
@@ -2891,6 +2917,11 @@ def apply(self, person_id, squeeze_factor):
             self.module, self, self.module.item_codes_lab_consumables, core='delivery_core',
             optional='delivery_optional')
 
+        # Log required equipment
+        self.EQUIPMENT.update({'Delivery set', 'Weighing scale', 'Stethoscope, foetal, monaural, Pinard, plastic',
+                               'Resuscitaire', 'Sphygmomanometer', 'Tray, emergency', 'Suction machine',
```

Comment:
Is the 'Sphygmomanometer' + 'Stethoscope' the same as 'Blood pressure machine'? If so, we may add 'Sphygmomanometer' as an alternative name to the 'Blood pressure machine' and replace these two items everywhere used.

Reply:
Hi @EvaJanouskova Yes that sounds right to me

Reply:
@EvaJanouskova i have documented the overlap of these two items in issue #1147. I would not just replace the stethoscope however as there are other important clinical indications for that piece of equipment beyond measuring BP with a sphyg.

Reply:
@joehcollins Yes happy with what you propose on cancers.

Reply:
thanks andrew!